src/ui/story.js

```diff
@@ -32,8 +33,37 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
                 // }
             ],
             author: "",
-            tags: []
+            tags: [],
+            published: false,
+            // These timestamps are intended to be in ISO-8601 format and UTC time
+            // https://en.wikipedia.org/wiki/ISO_8601
+            // E.g. "2020-04-22T16:50:44.324Z"
+            timestampCreated: "",
+            timestampLastModified: "",
+            timestampPublished: ""
+        },
+        invokers: {
```

Comment:
According to the discussion with @BlueSlug, `timestampLastModified` is created for addressing [SJRK-345 issue](https://issues.fluidproject.org/browse/SJRK-345) when we need to set up a rule to check staled stories. Since we haven't set up any rules in relation to this timestamp, it might be good to remove this code before it slips out of our mind. @BlueSlug will keep the commit hash to this code in SJRK-345 for the reference in the future when this code is useful.

Reply:
I have added a comment with the commit hash on the SJRK-345 Jira